API/cmd/migrate.go

```
@@ -0,0 +1,81 @@
+	Use:   "up",
+	Short: "Exécute toutes les migrations",
+	Run: func(cmd *cobra.Command, args []string) {
+		useful.RunMigrationCommand("up")
+	},
+}
+
+var migrateDownCmd = &cobra.Command{
+	Use:   "down",
+	Short: "Annule la dernière migration",
+	Run: func(cmd *cobra.Command, args []string) {
+		useful.RunMigrationCommand("down")
```

golangci-lint (1.62.2)

Error return value of `useful.RunMigrationCommand` is not checked

(errcheck)